pages/linux/mkisofs.md

```diff
@@ -0,0 +1,11 @@
+# mkisofs
+
+> Create ISO files from folders. Also aliased as `genisoimage`.
+
+- Create an ISO from a folder:
+
+`mkisofs -o {{filename.iso}} {{path/to/source/folder}}`
+
+- Set the disc label when creating an ISO:
+
+`mkisofs -o {{filename.iso}} {{path/to/source/folder}}`
```

Comment:
This example is same as the previous one. Perhaps you wanted to add some flag in this one.

Reply:
Oops! Good catch. Updating now.